pygeoapi/plugin.py

```diff
@@ -50,6 +50,7 @@
         'xarray': 'pygeoapi.provider.xarray_.XarrayProvider',
         'MVT': 'pygeoapi.provider.mvt.MVTProvider',
         'TinyDBCatalogue': 'pygeoapi.provider.tinydb_.TinyDBCatalogueProvider',
+        'Sensorthings': 'pygeoapi.provider.sensorthings.SensorthingsProvider',
```

Comment:
(global comment) should this be `SensorThings` (vs. `Sensorthings`)?

Reply:
Probably `SensorThings`